sqlglot/optimizer/simplify.py

```diff
@@ -1219,6 +1219,11 @@ def gen(expression: t.Any) -> str:
 }
 
 
+def _anonymous(e: exp.Anonymous) -> str:
+    name = e.this.upper() if isinstance(e.this, str) else e.sql()
```

Comment:
why are you calling e.sql() here? that ruins the purpose of gen

Reply:
Oversight, will fix - thanks

Reply:
done